### PR TITLE
SITE-6649 Ticket icon added for limit quantity option

### PR DESCRIPTION
**Pull request by msied**

### CODE REVIEW

File: app/components/Tickets.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  render() {
+    const classes = classnames('tilt-icon tilt-icon-tickets', this.props.className);
+    const textStyle1 = {
+      fontSize: '14px',
+      fontFamily: 'HelveticaNeueRoman',
+    };
+    const textStyle2 = {
+      fontSize: '10px',
+      fontFamily: 'HelveticaNeueRoman',
+    };
```
Comment (Jiert) [inline app/components/Tickets.js]: Are `textStyle1` and `textStyle2` used anywhere?

**Comment by Jiert** [comment]
I think you'll need to bump the version

**Comment by Jiert** [comment]
lgtm

File: app/components/Tickets.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const React = require('react');
+const classnames = require('classnames');
+
+module.exports = React.createClass({
```
Comment (tildedave) [inline app/components/Tickets.js]: can you use `defineSvg`?

File: app/stylesheets/Tickets.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.tilt-icon.tilt-icon-tickets {
+  max-width: 46px;
+  max-height: 46px;
+
+  .tilt-icon-main {
+    fill-rule: evenodd;
+    stroke-width: 1;
+    stroke: none;
+    .tilt-icon-inner-1{
```
Comment (tildedave) [inline app/stylesheets/Tickets.scss]: space before brace

File: app/stylesheets/Tickets.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    	fill: $tilt-blue;
+    }
+    .tilt-icon-inner-2{
```
Comment (tildedave) [inline app/stylesheets/Tickets.scss]: space

File: app/stylesheets/Tickets.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    	fill: #FFFFFF;
+    }
+    .tilt-icon-inner-3{
```
Comment (tildedave) [inline app/stylesheets/Tickets.scss]: space

File: app/stylesheets/Tickets.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+  .tilt-icon-main {
+    fill-rule: evenodd;
+    stroke-width: 1;
+    stroke: none;
+    .tilt-icon-inner-1{
+    	fill: $tilt-blue;
+    }
+    .tilt-icon-inner-2{
+    	fill: #FFFFFF;
```
Comment (tildedave) [inline app/stylesheets/Tickets.scss]: I prefer `fill: white` here

**Comment by msied** [comment]
ping

**Comment by tildedave** [comment]
lgtm A+ now

**Pull request merged**